Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -11,6 +11,10 @@ public class NetworkAnimator : NetworkBehaviour
         // configuration
         [SerializeField] Animator m_Animator;
         [SerializeField] uint m_ParameterSendBits;
+        // Note: not an object[] array because otherwise initialization is real annoying
+        int[] intShadowCopy;
+        float[] floatShadowCopy;
+        bool[] boolShadowCopy;
```

Comment:
maybe rename 'lastIntParameters' etc.?

Reply:
If you did this, resolve this conversation.